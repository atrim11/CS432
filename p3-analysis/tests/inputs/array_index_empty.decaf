int arr[3];

def int main() {
int b;
arr = 5;
b = arr;

return arr[add(1, 2)];
}

def int add(int a, int b) { return a + b; }
