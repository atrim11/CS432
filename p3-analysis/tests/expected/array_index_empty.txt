Array 'arr' accessed without index on line 5
Array 'arr' accessed without index on line 6
